core/integration/module_config_test.go

```diff
@@ -1421,7 +1429,13 @@ func (ModuleSuite) TestDaggerGitRefs(ctx context.Context, t *testctx.T) {
 
 			htmlURL, err := rootModSrc.AsGitSource().HTMLURL(ctx)
 			require.NoError(t, err)
-			require.Equal(t, fmt.Sprintf("https://%s/%s/%s", tc.expectedBaseHTMLURL, tc.expectedURLPathComponent, tc.gitTestRepoCommit), htmlURL)
+			var expectedURL string
+			if tc.expectedHost == "dev.azure.com" {
+				expectedURL = fmt.Sprintf("https://%s/%s/%s", tc.expectedBaseHTMLURL, tc.expectedURLPathComponent, tc.gitTestRepoCommit)
+			} else {
+				expectedURL = fmt.Sprintf("https://%s/%s/%s", tc.expectedBaseHTMLURL, tc.expectedURLPathComponent, tc.gitTestRepoCommit)
+			}
+			require.Equal(t, expectedURL, htmlURL)
```

Comment:
These are the same in both branches?

Instead of hardcoding `dev.azure.com` here, we could potentially have a `expectedPathPrefix` property or something in the test case, and the have that set to `?path=` for azure (and left empty for the others)?

Reply:
Done 🙏